source/containers.tex

```diff
@@ -11106,7 +11106,9 @@
 \pnum
 The exposition-only alias templates
 \exposid{iter\--value\--type}, \exposid{iter\--key\--type},
-\exposid{iter\--mapped\--type}, and \exposid{iter\--to\--alloc\--type}
+\exposid{iter\--mapped\--type}, \exposid{iter\--to\--alloc\--type},
+\exposid{range\--key\--type}, \exposid{iter\--mapped\--type},
+and \exposid{range\--to\--alloc\--type}
```

Comment:
Can we cut back on that jungle of hyphenation hints? Maybe we need one or two in strategic places here, but not 20.

Reply:
Absolutely.